### PR TITLE
Admin's dashboard

### PR DESCRIPTION
**Pull request by soniyadotp**

### CODE REVIEW

**Comment by manwar** [comment]
@soniyaprasad77 when you say admin login, you mean regular login page?

File: prototype/Frontend/admin-Sign-in.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Admin Login / Sign Up Form</title>
+    <link rel="stylesheet" href="css/admin-login.css" />
+  </head>
+  <body>
+    <!-- Header Section -->
+    <header>
+      <a href="index.html" id="logo"
+        ><img src="/assets/blogs/blog_img_6.jpeg" alt="Logo"
+      /></a>
```
Comment (manwar) [inline prototype/Frontend/admin-Sign-in.html]: Format is little odd, isn't it?

I would have expected something like:

          <a href="index.html" id="logo">
              Logo
          </a>

File: prototype/Frontend/admin-Sign-in.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            <div class="captcha-container">
+              <img id="captcha-image" src="#" alt="CAPTCHA Image" />
+
+              <div
+                style="
+                  display: flex;
+                  justify-content: space-evenly;
+                  align-items: center;
+                "
+              >
```
Comment (manwar) [inline prototype/Frontend/admin-Sign-in.html]: Again this should have been:

        <div  style="display: flex; justify-content: space-evenly; align-items: center;">

**Review by manwar — COMMENTED** [review]
Please check my comments.

**Comment by soniyadotp** [comment]
what should i add 

> @soniyaprasad77 when you say admin login, you mean regular login page?

How should I improve the page for admin, is there any specifications?

**Comment by soniyadotp** [comment]
> Please check my comments.

sure, I will improve the indentations.

**Comment by manwar** [comment]
@soniyaprasad77 You named the file admin login, so I was wondering if there would be another page for regular user login?
Let's keep it simple, one login page for both, regular user and admin user.

**Comment by soniyadotp** [comment]
> @soniyaprasad77 You named the file admin login, so I was wondering if there would be another page for regular user login? Let's keep it simple, one login page for both, regular user and admin user.

Alright., Simplicity is essential :)

**Comment by soniyadotp** [comment]
So I am closing this PR, Thank you for your suggestions.

**Comment by manwar** [comment]
@soniyaprasad77 you can keep the same pull request and add any changes. No need to create new pull request.

**Comment by soniyadotp** [comment]
Certainly! 

I would appreciate some clarification regarding user authentication within a web application. Given that administrators typically possess enhanced features compared to regular users, it seems logical to have distinct database schemas for administrators and students. However, I am curious about the feasibility of implementing a unified login mechanism on the same page, utilizing POST requests for both administrators and students. Could you kindly shed light on how such a scenario can be effectively managed, taking into consideration the differing functionalities associated with these user roles? Your insights would be immensely valuable.

**Comment by manwar** [comment]
@soniyaprasad77 authentication for both admin and regular user would be same. Both go through the same process but once logged in, admin and regular user would be redirected to their respective dashboard.
Does this make sense?

**Comment by manwar** [comment]
@soniyaprasad77 did you make the changes I suggested earlier? Is this ready to be merged?

**Comment by soniyadotp** [comment]
Actually, as you suggested, we are not going for a different login page for the admins, so I started working on the admin dashboard, I am still working on it, I will update it soon.

**Comment by manwar** [comment]
@soniyaprasad77 sorry for the confusion, there is going to be one login for both members and admin, i.e. both would use the same login page.

After successfully logged in, the login user would be re-directed to their own dashboard i.e. member would go member dashboard and admin would go to admin dashboard.

Does this make sense?

**Comment by soniyadotp** [comment]
Yes.. it's make sense, and we do already have a login page. So I am working on Admins dashboard

**Pull request closed without merging**